wiki/Community/Project_Loved/en.md

```diff
@@ -23,32 +23,29 @@ Creators of said beatmaps are also able to request them to be removed from the L
 
 There are some very minimal criteria that need to be met for beatmaps to be nominated for Loved. Each game mode has its own specific criteria for Loved beatmap nomination and priority, but all modes follow these rules:
 
-- It is not in the Qualified, Ranked, or Approved categories
+- It abides by the [beatmap submission rules](/wiki/Rules#beatmap-submission-rules)
+- It is not in the [Ranked](/wiki/Beatmap/Category#ranked), [Approved](/wiki/Beatmap/Category#approved), or [Qualified](/wiki/Beatmap/Category#qualified) categories
 - Its creator does not disapprove of it being Loved
-- Its creator is not restricted
+  - If any guest contributors disapprove of their part being Loved, their part will be excluded from Loved by appropriate means. In some cases, such as [collabs](/wiki/Beatmap/Beatmap_collaborations), this may make the entire beatmap ineligible for Loved.
+- Its creator is not [restricted](/wiki/Help_centre/Account_restrictions)
 - At least one difficulty can be passed
+- Every nominated difficulty has a [drain time](/wiki/Beatmap/Drain_time) of at least 20 seconds
 
-Although many Pending and Work-in-progress beatmaps meet these criteria, few are voted into the Loved category due to the more selective nomination and voting requirements.
+Although many [Work in Progress and Pending](/wiki/Beatmap/Category#wip-and-pending) beatmaps meet these criteria, few are voted into the Loved category due to the selective nature of the captains' picks, and the high community approval requirement by voting.
```

Comment:
hmm maybe this could be simplified to newer maps, feel like people make minor changes to maps all the time to keep them in pending for attention

Reply:
i dont think it was meant to imply anything about the age of the map, it's just missing "graveyard" for some reason, i will fix that instead